frontend/src/pages/dashboard/components/user-profile-info/styles.module.scss

```diff
@@ -0,0 +1,45 @@
+.container {
+  max-width: 350px;
+  height: 100%;
+  background: linear-gradient(
+    to bottom,
+    var(--transparent) 0,
+    var(--blue-100) 100%
+  );
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  height: 10%;
```

Comment:
We should avoid relative values for layout

Reply:
@Liza-Veis so should i use px instead of percentage?

Reply:
which element is it?